src/test/regress/expected/isolation_rebalancer_deferred_drop.out

```diff
@@ -55,6 +55,28 @@ step s1-commit:
     COMMIT;
 
 
+starting permutation: s1-begin s1-move-placement s2-move-placement s1-commit
+step s1-begin:
+    BEGIN;
+
+step s1-move-placement:
+        SET citus.defer_drop_after_shard_move TO ON;
+     SELECT master_move_shard_placement((SELECT * FROM selected_shard), 'localhost', 57637, 'localhost', 57638);
+
+master_move_shard_placement
+
+
+step s2-move-placement:
+        SET citus.defer_drop_after_shard_move TO ON;
+        SET citus.force_disk_available = 20;
+     SELECT master_move_shard_placement((SELECT * FROM selected_shard), 'localhost', 57637, 'localhost', 57638);
+ <waiting ...>
+step s1-commit: 
+    COMMIT;
+
+step s2-move-placement: <... completed>
```

Comment:
is this test giving the expected output? I guess the previous step should have been from 57638 to 57637 ?

And, why is this test an isolation test? I couldn't follow